setup.cfg

```diff
@@ -17,7 +17,7 @@ packages = find:
 setup_requires =
     setuptools_scm
 install_requires =
-    click>=7.0
+    click~=7.0
```

Comment:
As we discussed in Slack, Click v8.0.0 was released yesterday and some tests does not work with the version